debian/control

```diff
@@ -19,8 +19,10 @@ Depends:
  ${misc:Depends},
  ${python3:Depends},
  ${shlibs:Depends},
-# Explicitly included, as dh_shlibdeps is overridden
- libc6-dev,
+# Required to avoid conflict with allowing plugdev group to mount disks
+# https://github.com/RPi-Distro/raspberrypi-sys-mods/
+# commit #59fd502bd4fcf8f5f646ba6c7ad59c8756c3094e
+ raspberrypi-sys-mods (>= 20210706),
```

Comment:
```suggestion
# Required to avoid conflict with config file allowing plugdev group
# to mount disks
#
# https://github.com/RPi-Distro/raspberrypi-sys-mods/
# commit #59fd502bd4fcf8f5f646ba6c7ad59c8756c3094e
 raspberrypi-sys-mods (>= 20210706),
```